packages/p2p/src/components/verification/verification.tsx

```diff
@@ -37,15 +38,21 @@ const Verification = ({ should_wrap }) => {
         {
             content: general_store.nickname || <Localize i18n_default_text='Choose your nickname' />,
             status: general_store.nickname ? 'done' : 'action',
-            onClick: general_store.nickname ? () => {} : general_store.toggleNicknamePopup,
+            onClick: general_store.nickname
+                ? () => {
+                      //do nothing
+                  }
+                : general_store.toggleNicknamePopup,
         },
         {
             content: general_store.poiStatusText(general_store.poi_status),
             is_disabled: general_store.poi_status !== 'verified' && !general_store.nickname,
             status: general_store.poi_status === 'verified' ? 'done' : 'action',
             onClick:
                 general_store.poi_status === 'verified'
-                    ? () => {}
+                    ? () => {
+                          //do nothing
```

Comment:
`nit` tbh i don't think we need this comment here since we know this function won't return or do anything

Reply:
it gives a eslint error if given an empty function